File: C/prime_number.c
#include <stdio.h>

int main() {
    int n, i, flag = 0;

    printf("Enter the Number to check Prime: ");
    scanf("%d", &n);

    if (n <= 1) {
        printf("Number is not Prime.\n");
        return 0;
    }

    for (i = 2; i <= n / 2; i++) {
        if (n % i == 0) {
            printf("Number is not Prime.\n");
            flag = 1;
            break;
        }
    }

    if (flag == 0)
        printf("Number is Prime.\n");

    return 0;
}

File: JavaScript/prime_number.js
const prompt = require('prompt-sync')();

const n = parseInt(prompt("Enter the Number to check Prime: "));
let flag = 0;

if (n <= 1) {
    console.log("Number is not Prime.");
} else {
    for (let i = 2; i <= n / 2; i++) {
        if (n % i === 0) {
            console.log("Number is not Prime.");
            flag = 1;
            break;
        }
    }

    if (flag === 0)
        console.log("Number is Prime.");
}
